hawkbit-repository/hawkbit-repository-test/pom.xml

```diff
@@ -35,6 +35,10 @@
          <version>${project.version}</version>
       </dependency>
       <dependency>
+         <groupId>org.springframework.boot</groupId>
```

Comment:
This dependency is not needed here. There is already a transitve dependency to `spring-boot-starter-logging` provided by `spring-cloud-stream-binder-test`